src/common/asb/Asb.c

```diff
@@ -604,6 +608,14 @@ void AsbInitialize(void* log)
     {
         OsConfigLogError(log, "AsbInitialize: failed to allocate memory");
     }
+
+    if (false == FileExists(g_etcFstabCopy))
```

Comment:
What is the reason for not doing `!FileExists(g_etcFstabCopy) && !MakeFileBackupCopy(g_etcFstab, g_etcFstabCopy, log)`? Is this style consistent all throughout?

Reply:
To be fully explicit here, as rest of this function. For Boolean values this is the best way, followed by no comparison for true and then ! for false.